xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -997,4 +1000,36 @@ void CGUIFontTTFBase::EmboldenGlyph(FT_GlyphSlot slot)
   slot->metrics.vertAdvance  += dy;
 }
 
+// Lighten code - original taken from freetype2 (ftsynth.c)  
+void CGUIFontTTFBase::LightenGlyph(FT_GlyphSlot slot)
+{
+  if (slot->format != FT_GLYPH_FORMAT_OUTLINE)
+    return;
+
+  /* some reasonable strength */
+  FT_Pos strength = FT_MulFix(m_face->units_per_EM,
+                              m_face->size->metrics.y_scale) / -48;
```

Comment:
@phil65 we can take care of that afterwards as it doesn't change the interface skinners interact with.